examples/rapids/mlflow_project/notebooks/rapids_mlflow.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
The notebook is great! Some questions/comments:
* Why is the output-helper necessary for displaying the output of `mlflow models serve`? I guess IPython won't display server output anywhere otherwise? For simplicity, could we instead encourage the user to run `mlflow models serve` via the CLI in a separate terminal window after they've trained their model in the notebook?
* Similar to the other example, I wonder if we can actually log to a local sqlite file in order to use model registry features (as usage of SQL-backed storage and model registry is the encouraged best-practice), rather than working around it

Reply:
We could direct the user to run the model serve call in a separate terminal; however, I think they already get that workflow by following the CLI instructions. This just provides a more self-contained example, fully self-contained after migrating to the sqlite storage solution, since it lets us look up the artifact uri.

All examples will use sqllite in the next PR update.

Reply:
@drobison00 got it, I see where you're coming from re striving for a self-contained example. However, IMO it'd still be better to recommend using the CLI to serve models - in our examples, we want to encourage best practices for using RAPIDS and MLflow, and currently, the recommended & well-supported way to serve models is to invoke the `mlflow models serve` CLI via terminal, rather than attempt to run it as a subprocess and process the output via a helper thread etc. It'd also make the notebook shorter, which is always nice to have in an example :)

Reply:
Let me know if that makes sense btw, happy to discuss further!